content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -156,24 +170,30 @@ To allow signing a CertificateSigningRequest:
 
 {{< codenew file="access/certificate-signing-request/clusterrole-sign.yaml" >}}
 
-## Normal User
+## Normal user
 
-A few steps are required in order to get normal user to be able to authenticate and invoke an API. First, this user must have certificate issued by the Kubernetes Cluster, and then present that Certificate to the API call as the Certificate Header or through the kubectl.
+A few steps are required in order to get normal user to be able to
+authenticate and invoke an API. First, this user must have certificate issued
+by the Kubernetes cluster, and then present that Certificate to the API call
+as the Certificate Header or through the kubectl.
 
-### Create Private Key
+### Create private key
 
-The following scripts show how to generate PKI private key and CSR. It is important to set CN and O attribute of the CSR. CN is the name of the user and O is the group that this user will belong to. You can refer to [RBAC](/docs/reference/access-authn-authz/rbac/) for standard groups.
+The following scripts show how to generate PKI private key and CSR. It is
+important to set CN and O attribute of the CSR. CN is the name of the user and
+O is the group that this user will belong to. You can refer to
+[RBAC](/docs/reference/access-authn-authz/rbac/) for standard groups.
 
-```
+```shell
 openssl genrsa -out john.key 2048
 openssl req -new -key john.key -out john.csr
 ```
 
-### Create Certificate Request Kubernetes Object
+### Create certificate signing request
```

Comment:
nit: Line 194, CertificateSigningRequest is a resource. The heading is using lowercase.
Should the resource be formatted as code?

Reply:
both 'certificate signing request' (general term) and 'CertificateSigningRequest' (resource name) are acceptable, right?